src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -383,6 +399,20 @@ public String getLocation() {
       return location;
     }
 
+    @Nullable
+    public String getContentType() {
+      if (!Strings.isNullOrEmpty(contentType)) {
+        if (contentType.equals(CONTENT_TYPE_OTHER)) {
+          if (Strings.isNullOrEmpty(customContentType)) {
```

Comment:
This means if `customContentType` is a macro, we use default content type?

Reply:
Same in here as well, if the value of customContentType is a macro, the value that was specified by the user is being used.